lib/handlers/assembly-documentation.ts

```diff
@@ -36,7 +36,8 @@ const onDocumentationProviderRequest = (
 ) => {
     // If the request had no opcode parameter, we should fail. This assumes
     // no assembly language has a __unknown_opcode instruction.
-    const instruction = (request.params.opcode || '__UNKNOWN_OPCODE').toUpperCase();
+    const instruction = (request.params.opcode || '__UNKNOWN_OPCODE');
+    // PTX is in lowercase, SASS is in uppercase
```

Comment:
I would be surprised if this did not break something. Surely we have some compiler somewhere that outputs x86 in all caps or something. (That's kind of the recurring problem with compiler explorer, there is always some compiler doing something a little different).

Would it be possible to split the assembly docs for sass/ptx and then have the front end choose which instruction set to use?

Reply:
I have checked all the asm docs code, they all do `.toUpperCase()` in `lib/asm-docs/generated/asm-docs-*.ts`, except for `arm` which I have added in `lib/asm-docs/arm.ts`.

I choose this method because I couldn't figure out how to implement PTX and SASS separately. I will try to figure that out.

Reply:
I have figured out how to split the assembly docs for sass/ptx. Do you still want the removal of `.toUpperCase()` reverted?

Reply:
Thanks for the new changes, let's go ahead and remove toUpperCase I think here just to avoid any surprises in the future

Reply:
done!